### PR TITLE
add CITATION.cff file

### LINKED ISSUE #394: create CFF citation file
Add a [citation CIFF file](https://citation-file-format.github.io/) to the repo so that zenodo picks up all authors. We need this for the final JOSS paper acceptance https://github.com/openjournals/joss-reviews/issues/6934#issuecomment-2342882680.
- [x] all authors and contributors
- [x] add the JOSS paper #328

After adding the CIFF file we need to make another patch level release. 

See https://github.com/openjournals/joss-reviews/issues/6934#issuecomment-2342882680 for background:

> One can tune the metadata in Zenodo using a CIFF file: https://citation-file-format.github.io/. You can see an example for one of my packages [here](https://github.com/pyscal/atomRDF/blob/main/CITATION.cff). You would need to make another release after, probably with updated version. There is also contributer role available, so if you want to credit other people who are not in the paper, that could be done.

### PR DESCRIPTION
**Pull request by orbeckst**
- close #394 
- add all AUTHORS with ORCIDs and affiliation (as far as they are confirmed)
  - order based on AUTHORS
- add alchemlyb PIs
- add contributors that were listed in JOSS paper acknowledgements (PR #328) but had NOT been listed in AUTHORS (inserted in chronological order)
- only add emails for maintainers/PIs
- add Software Heritage Identifier
- add paper citation section for MBAR, decorrelation, and JOSS paper #328 

-----

- add one new contributor (from recent PR #381 ) to AUTHORS and CITATIONS.cff

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - email: dotsdl@gmail.com
+    given-names: David L.
+    family-names: Dotson
+    orcid: 'https://orcid.org/0000-0001-5879-2942'
+    affiliation: Datryllic LLC
```
Comment (orbeckst) [inline CITATION.cff:16]: @dotsdl are you ok with this entry, in particular your email address here?

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    family-names: Beckstein
+    email: obeckste@asu.edu
+    affiliation: Arizona State University
+    orcid: 'https://orcid.org/0000-0003-1340-0831'
+  - given-names: Shuai
+    family-names: Liu
+    orcid: 'https://orcid.org/0000-0002-8632-633X'
+    affiliation: Silicon Therapeutics LLC
+  - given-names: Travis
+    family-names: Jensen
```
Comment (orbeckst) [inline CITATION.cff:31]: @trje3733 do you want to add any additional information here?

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  - given-names: Bryce
+    family-names: Allen
+    orcid: 'https://orcid.org/0000-0002-0804-8127'
+    affiliation: Differentiated Therapeutics
+  - given-names: Dominik
+    family-names: Wille
+    affiliation: Freie Universität Berlin
```
Comment (orbeckst) [inline CITATION.cff:38]: @harlor do you want to add any additional information here?

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    affiliation: Pacific Northwest National Laboratory
+  - given-names: Mohammad Soroush
+    family-names: Barhaghi
+    orcid: 'https://orcid.org/0000-0001-8226-7347'
+    affiliation: Wayne State University
+  - given-names: Zhiyi
+    family-names: Wu
+    affiliation: Exscientia plc
+    orcid: 'https://orcid.org/0000-0002-7615-7851'
+    email: william@zhiyiwu.me
```
Comment (orbeckst) [inline CITATION.cff:55]: @xiki-tempula are you ok with this information?

**Review by orbeckst — COMMENTED** [review]
Please let me know if you want to make any changes to your information.

**Comment by orbeckst** [comment]
Note to self: merge (no squash merge) so that individual commits remain visible in the history in case there are questions about decisions.

File: CITATION.cff (hunk 4 above)
Comment (xiki-tempula) [reply CITATION.cff:55]: Yes, I'm happy.

**Review by xiki-tempula — APPROVED** [review]

**Comment by hl2500** [comment]
Hello @orbeckst, thank you for creating the cff file. It seems my affiliation in the file is not correct. It was listed as UC Irvine rather than the UNC one. I'm wondering if it's possible to change? Thanks.

File: CITATION.cff (hunk 4 above)
Comment (orbeckst) [reply CITATION.cff:55]: thanks

File: CITATION.cff [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+    orcid: 'https://orcid.org/0000-0002-0271-2760'
+    affiliation: Arizona State University
+  - given-names: Jan
+    family-names: Janssen
+    affiliation: Max-Planck-Institut for Sustainable Materials
+    orcid: 'https://orcid.org/0000-0001-9948-7119'
+  - given-names: Haoxi
+    family-names: Li
+    orcid: 'https://orcid.org/0009-0004-8369-1042'
+    affiliation: University of North Carolina
```
Comment (orbeckst) [inline CITATION.cff:95]: @hl2500 affiliation fixed

File: CITATION.cff [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+      orcid: 'https://orcid.org/0000-0002-9749-8866'
+      affiliation: Arizona State University
+    - given-names: Hyungro
+      family-names: Lee
+      orcid: 'https://orcid.org/0000-0002-4221-7094'
+      affiliation: Pacific Northwest National Laboratory
+    - given-names: Haoxi
+      family-names: Li
+      orcid: 'https://orcid.org/0009-0004-8369-1042'
+      affiliation: University of North Carolina
```
Comment (orbeckst) [inline CITATION.cff:181]: @hl2500 affiliation fixed

**Review by orbeckst — COMMENTED** [review]
@hl2500 thanks for checking, apologies for my mistake. I fixed your affiliation (see below).

**Comment by orbeckst** [comment]
@dotsdl @davidlmobley @mrshirts could you please have a look at the author list here by the end of Monday 9/16? I would like to finish the JOSS paper and this is the very last thing that's needed.

If you need more time please let me know. 

If I don't hear anything else by Tuesday morning I'll take this as tacit agreement and we'll make a release #396 and finish the paper #71.

File: CITATION.cff (hunk 1 above)
Comment (dotsdl) [reply CITATION.cff:16]: Yes, this email works for this purpose! Thanks so much for leading this!

Comment (dotsdl) [reply CITATION.cff:16]: Actually, could we swap this out for `david@datryllic.com` instead? Thank you!

File: CITATION.cff [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: alchemlyb
+message: >-
+  If you use this software, please cite it using the
+  preferred citation together with any other references.
+type: software
+authors:
+  - email: dotsdl@gmail.com
```
Comment (dotsdl) [inline CITATION.cff:12]:
```suggestion
  - email: david@datryllic.com
```

File: CITATION.cff (hunk 1 above)
Comment (orbeckst) [reply CITATION.cff]: of course! — done with your suggestion

**Comment by dotsdl** [comment]
LGTM!

**Comment by mrshirts** [comment]
This looks good to me!

**Comment by orbeckst** [comment]
@xiki-tempula given that you merged PR #381 please update on this PR
- CHANGELOG
- add @jaclark5 to the CITATION.cff
- update issue #396 (we are now going to make a 2.4.0 release for the JOSS paper)

Thanks.

File: CITATION.cff [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+    family-names: Mobley
+    orcid: 'https://orcid.org/0000-0002-1083-5533'
+    affiliation: University of California Irvine    
+  - given-names: Michael R.
+    family-names: Shirts
+    orcid: 'https://orcid.org/0000-0003-3249-1097'
+    affiliation: University of Colorado Boulder
+  - given-names: Jennifer A.
+    family-names: Clark
+    affiliation: National Institute of Standards and Technology
```
Comment (orbeckst) [inline CITATION.cff:110]: @jaclark5 is this information correct (name, affiliation)

Do you have an ORCID that we could add?

Comment (jaclark5) [reply CITATION.cff:110]: Yes it's correct. https://orcid.org/0000-0003-4897-5651

Comment (orbeckst) [reply CITATION.cff]: thanks, updated

**Comment by orbeckst** [comment]
* close #394
* add all AUTHORS with ORCIDs and affiliation (as far as they are confirmed); order based on AUTHORS
* add alchemlyb PIs
* add contributors that were listed in JOSS paper acknowledgements 
  (PR #328) but had NOT been listed in AUTHORS (inserted in chronological order)
* only add emails for maintainers/PIs
* add Software Heritage Identifier
* add paper citation section for MBAR, decorrelation, and preliminary citation for the 
  JOSS paper #71
* add one new contributor (from recent PR #381 ) to AUTHORS and CITATIONS.cff

**Pull request merged**